### ISSUE TITLE
Musescore 4.4.0 AppImage won't open

### ISSUE THREAD

**Issue by chunga2020** [musescore/MuseScore#24228]
### Issue type

Opening/saving issue

### Description with steps to reproduce

Trying to open the new 4.4.0 AppImage fails.  Error output indicates a missing OpenGL `.so`

1. Download the AppImage
2. `chmod +x <the file>`
3. `./MuseScore-Studio-4.4.0.AppImage`; I stripped off the rest of the filename

### Supporting files, videos and screenshots

```
$ ./MuseScore-Studio-4.4.0.AppImage
/lib/x86_64-linux-gnu/libjack.so.0
/lib/x86_64-linux-gnu/libnss3.so
/lib/x86_64-linux-gnu/libwayland-client.so.0
/tmp/.mount_MuseScIbiNzl/bin/mscore4portablenightly: error while loading shared libraries: libOpenGL.so.0: cannot open shared object file: No such file or directory
```

### What is the latest version of MuseScore Studio where this issue is present?

4.4.0

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Pop!_OS 24.04 LTS

### Additional context

I can run the 4.3.2 AppImage just fine.  I tried both the main 4.4.0 release and the nightly release, and both fail.

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by cbjeukendrup** [comment]
Hello! We have not one but _two_ potential fixes for this:
1. https://github.com/musescore/MuseScore/actions/runs/10817184068
2. https://github.com/musescore/MuseScore/actions/runs/10819513145 (ready in ca. 25 minutes from the time of writing)

You can download these and test these AppImages _in that order_ (by clicking on the download link under 'Artifacts'). It would be great if you could let us know if one of them fixes the problem!

**Comment by chunga2020** [comment]
@cbjeukendrup So I tried both of the potential fixes that you suggested.  Neither worked off the bat.  I ended up doing `apt install libopengl0`, which finally fixed it.  Not ideal, but it works, I guess

**Comment by cbjeukendrup** [comment]
@chunga2020 Thanks for letting us know. Here is another attempt: https://github.com/musescore/MuseScore/actions/runs/10850359655?pr=24696

**Comment by chunga2020** [comment]
@cbjeukendrup did an `apt remove libopengl0` and tried this one, and it works!

**Comment by cbjeukendrup** [comment]
Great! The fix will be included either in 4.4.2 or 4.4.3, I'm currently not sure which one.
